python/MooseDocs/extensions/layout.py

```diff
@@ -89,6 +92,19 @@ def createToken(self, parent, info, page):
 
         return col
 
+class ScrollCommand(command.CommandComponent):
```

Comment:
I am not sold on this command, can you pull this stuff out into another branch/PR? Don't do that until I push up my other changes.

Reply:
@aeslaughter Yes. Just let me know when you are done.